### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by rittik-ghosh**
Hi 

I have added the cart page. Please review & let me know of any improvements.

cart

### CODE REVIEW

File: Stylesheets/cart.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.quantity button {
+    margin: 0 0.5rem;
+}
+
+.display-count {
+    border: 1px solid var(--st-black);
+    border-radius: 5px;
+    width: 20px;
```
Comment (aatishpandey) [inline Stylesheets/cart.css]: you can use rem here

File: Stylesheets/product.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+.flex-spacebetween{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* card wishlist */
+.wishlist-badge{
+    width: 25px;
```
Comment (aatishpandey) [inline Stylesheets/product.css:22]: here also

**Review by aatishpandey — APPROVED** [review]
Looks good. Suggested some minor changes

**Review by vaibhav-mahalle — APPROVED** [review]
everything looks good. Nice and clean code structure. Keep up the good work.

File: Stylesheets/cart.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+.checkout-container{
+padding:1rem;
+}
+
+.footer-container-cart {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
```
Comment (Gautam2305) [inline Stylesheets/cart.css:41]: These are the default values and you don't need to mention them specifically......

**Review by Gautam2305 — COMMENTED** [review]
suggested a change...do look into that

File: Stylesheets/cart.css (hunk 3 above)
Comment (rittik-ghosh) [reply Stylesheets/cart.css:41]: the footer would change if I am not putting those values.
that is why i have put that

File: Stylesheets/cart.css (hunk 1 above)
Comment (rittik-ghosh) [reply Stylesheets/cart.css]: done thanks for pointing it out

File: Stylesheets/product.css (hunk 2 above)
Comment (rittik-ghosh) [reply Stylesheets/product.css:22]: done

**Pull request merged**